jekyll/_cci2/release/configure-your-kubernetes-components.adoc

```diff
@@ -0,0 +1,223 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Configure your Kubernetes components - Open preview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Tutorial outlining how to configure your Kubernetes components for visibility and control on the releases dashboard.
+:icons: font
+:experimental:
+
+NOTE: CircleCI releases is currently in **open preview**. During open preview release features are available for free. Charges will be associated with release features once they are generally available.
+
+The steps outlined on this page guide you to configure your Kubernetes resources for integration with your release environment.
+
+[#introduction]
+== Introduction
+
+In this tutorial you will configure your Kubernetes components, adding labels and annotations to enable visibility and control over your deployments, and access to the following features:
+
+* Restore a version
+* Scale a version
+* Restart a version
+* Retry a release (when using Argo Rollouts)
+* Promote a release (when using Argo Rollouts)
+* Cancel a release (when using Argo Rollouts)
+
+[#prerequisites]
+== Prerequisites
+
+Before following the steps below, ensure the release environment has been set up successfully. Refer to the xref:set-up-a-release-environment#[Set up a release environment] page.
+
+[#add-required-labels]
+== 1. Add required labels to your Deployment/Rollout
+
+To enable your release to show up on the releases dashboard in the CircleCI web app, add the following labels to your Kubernetes Deployment or Argo Rollout:
+
+* Specify the `app` and `version` label in the Kubernetes object `Metadata.Labels`
+* Specify the `app` and `version` label in the Kubernetes object `Spec.Template.Metadata.Labels`
+
+For example:
+
+[,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: example-deployment
+    version: 1.0.0
+  name: example-deployment
+  namespace: default
+spec:
+  selector:
+    matchLabels:
+      app: example-deployment
+  template:
+    metadata:
+      labels:
+        app: example-deployment
+        version: 1.0.0
+----
+
```

Comment:
~ Perhaps worth adding a note here to specifically call out not adding `version` to the selector labels since those can't be changed later, and `version` is meant to change frequently.